exporters/ibmmq/e2e/e2e_spec.yml

```diff
@@ -36,4 +36,9 @@ scenarios:
       metrics:
         - source: "ibmmq.yml"
           except_entities: []
-          except_metrics: []
+          except_metrics: 
+            # TODO Add a topic pub/sub noise generator to the existing DEV.BASE.TOPIC topic.
```

Comment:
can we create an issue and link it here?

Reply:
make sense!